packages/drivers/odsp-driver-definitions/api-report/odsp-driver-definitions.legacy.alpha.api.md

```diff
@@ -241,6 +241,7 @@ export interface ShareLinkInfoType {
         error?: any;
         shareId?: string;
     };
+    doNonDurableRedeem?: boolean;
```

Comment:
Nit: is this standard terminology? If not, what about `isNonDurable` or `isRedemptionNonDurable`?